README.md

```diff
@@ -30,6 +30,54 @@ Any of the extensions listed below can be mixed into a class by `include`-ing `H
 
 ### Coercion
 
+Three systems are available for coercion: ActiveModel types, dry-types, and hashie's builtin types. The use of a specified coercion system is determined by exactly which Coercion module you include in your Hashie class. Including the default `Hashie::Extensions::Coercion` module will use Hashie's builtin type system.
+
+### Coercion using ActiveModel types
+
+To use ActiveModel's type coercion system, you must add version 5 of ActiveModel to your Gemfile. If you are using Rails, activemodel is included, but you must be using Rail's version 5 in order to use this. If you are on running a lower version of Rails, you will not be able to use ActiveModel's type coercion system due to Ruby's inability to use different Gem versions in the same application.
+
+```ruby
+gem 'activemodel', '~> 5.x'
+```
+
+Then include the `active_model` Coercion module into your Hashie class:
+
+```ruby
+class Tweet < Hash
+  include Hashie::Extensions::Coercion.active_model
```

Comment:
I actually do not find it weird and I actually think it makes more sense than a constant because it conveys that this returns an anonymous module. Plenty of gems do use this style, such as dry-types: http://dry-rb.org/gems/dry-types/including-types/ and virtus: https://github.com/solnic/virtus#using-virtus-with-classes.

That is the argument for it and why I set it up this way. Let me know if you may have been persuaded :-). If not, no problem and I'll update this to use constants.